types/selenium-webdriver/test/bidi/logInspector.ts

```diff
@@ -0,0 +1,15 @@
+import * as webdriver from "selenium-webdriver";
+import LogInspector from "selenium-webdriver/bidi/logInspector";
+
+function TestLogInspector() {
+    const driver: webdriver.WebDriver = new webdriver.Builder().build();
+    const logInspector = LogInspector(driver);
+    let lastLog;
+    logInspector.onLog(log => {
+        lastLog = log;
+    })
+    driver.executeScript("console.log('hello!')");
+    if (lastLog.text !== 'hello!') {
+        throw new Error(`Expected log text: ${lastLog.text} to equal 'hello!'`)
```

Comment:
```suggestion
    if (lastLog?.text !== 'hello!') {
        throw new Error(`Expected log text: ${lastLog?.text} to equal 'hello!'`)
```
`onLog` isn't guaranteed to run, so `lastLog` isn't guaranteed to have a value.

Reply:
@SupernaviX - I agree, so I reworked this to instead flip a complete boolean and throw an error if it isn't flipped in a second and a half. `lastLog` was kinda clunky for this anyway